setup.py

```diff
@@ -31,11 +31,15 @@
 #   XLA_CUDA=0
 #     build the xla/xrt client with CUDA enabled
 #
+#   BUNDLE_LIBTPU=1
```

Comment:
current default is 0, so we should put `BUNDLE_LIBTPU=0` here

Reply:
Oops, fixed.